tests/error/linear_errors/field_copy2.err

```diff
@@ -0,0 +1,7 @@
+Guppy compilation failed. Error in file $FILE:17
+
+15:    @guppy(module)
+16:    def foo(s: MyStruct) -> tuple[MyStruct, qubit]:
+17:        return s, s.q
+                     ^^^
+GuppyError: Field `s.q` with linear type `qubit` was already used (at 17:11)
```

Comment:
Can we record the use site of linear variables and point to where they were used when we give these error messages? I guess this requires upgrading the error framework to handle multiple locations

Reply:
We're already tracking the point where the variable has been used (that's the `(at 17:11)` part in the message). But it would be better tyo show this with a separate span 👍

Added to #306